modules/exploits/multi/php/wordpress_duplicator.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'            => 'Snapcreek Duplicator WordPress plugin installer.php code injection',
+      'Description'     => %q{
+        When the WordPress plugin Duplicator restores a backup, it leaves dangerous files in the filesystem such as
+        installer.php and installer-backup.php. These files allow anyone to call a function that overwrite the wp-config.php file
+        AND this function does not sanitize POST parameters before inserting them inside the wp-config.php file, leading to
+        arbitrary PHP code execution.
+        WARNING: This exploit WILL break the wp-config.php file. If possible try to restore backups of the configuration after the exploit
+        to make the WordPress site work again.
+      },
+      'Author'          => [ 'Julien Legras <julien.legras@synacktiv.com>', 'Thomas Chauchefoin <thomas.chauchefoin@synacktiv.com>' ],
```

Comment:
multi line this, see https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/bsd/finger/morris_fingerd_bof.rb#L23